utils/githooks/pre-commit.d/71-flake.sh

```diff
@@ -47,12 +45,17 @@ if [ "$BRANCH" = "origin/master" ]; then
     echo "  Checking tree"
     flake8 --statistics
 else
-
-    # shellcheck disable=SC1091
+    rc=0
 
     # non-scons
-    git diff "$TARGET" -U10 | flake8 --config .flake8 --diff
+    if ! echo "$py_files" | grep -vi scons | xargs flake8 --config .flake8; then
```

Comment:
@brianjmurrell The idea is instead of using the `--diff` option, we pipe the files to flake agnostically

Reply:
Ahhh.  I see.